data/animations/animations.inc

```diff
@@ -992,3 +992,7 @@
 .include "data/animations/anims/anim_countdown_start.inc"
 .include "data/animations/anims/anim_1131.inc"
 .include "data/animations/anims/anim_1132.inc"
+
+.if NON_MATCHING
+.include "data/animations/anims/anim_logos.inc"
```

Comment:
Minor, but could you consider putting these in a different directory to also differentiate them?

Reply:
I don't know, I'd rather not.
I think having everything in one place is more "visible" to users.
And it'll all be automated down the line, anyway.

Reply:
Sounds fine, if we want to include these under `ENHANCEMENTS` or something that's also fine in the future :)

Reply:
Ok, I understand I just don't want it to get confusing

Reply:
You also need to change this flag to `ENABLE_DECOMP_CREDITS`